<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSetting</class>
 <widget class="QWidget" name="CameraSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraSetting</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>750</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>相机设置</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>硬件设置</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>启用硬件触发</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>记录图像数据</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>硬触发实时显示</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>保存Matlab格式</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_captureMode">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>28</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>图像采集模式：</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_crossEnable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>中心线对齐</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_crossGapWidth">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_imgDLP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>341</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_imgDLP2Save">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>【DLP1】文件路径</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_imgDLP1Select">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_imgDLP2Select">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_imgDLP2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_imgDLP1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>【DLP2】文件路径</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_imgDLP1Save">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>351</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>视觉通用设置</string>
     </property>
     <widget class="QCheckBox" name="checkBox_LogAllCase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>启用错误日志</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PRInit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>系统初始化</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PRRelease">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>系统释放</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PRClearRecord">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>261</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>清空所有 Record</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_AutoClearRecord">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>自动清空前面Record</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_LogType">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>日志类型：</string>
      </property>
      <widget class="QRadioButton" name="radioButton_logFailCase">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>141</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>记录错误日志</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_logAllCase">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>141</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>记录所有日志</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>相机标定</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>601</height>
      </rect>
     </property>
     <property name="title">
      <string>相机标定</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>34</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Row 数目： </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>74</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Col 数目：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>114</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>物理距离：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>313</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X Resolution</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>343</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y Resolution</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>230</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Calibration</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>113</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>匹配分数：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>230</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>3D View</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>380</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>单点误差：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>pixel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>310</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>um/pixel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>um/pixel</string>
      </property>
     </widget>
     <zorder>line_3</zorder>
     <zorder>label</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>lineEdit_5</zorder>
     <zorder>lineEdit_6</zorder>
     <zorder>line_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_6</zorder>
     <zorder>lineEdit_7</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>lineEdit_8</zorder>
     <zorder>label_19</zorder>
     <zorder>label_20</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>10</y>
     <width>760</width>
     <height>750</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
